apps/api/src/app/inbox/utils/types.ts

```diff
@@ -8,9 +8,15 @@ export type Subscriber = {
   subscriberId: string;
 };
 
+type Redirect = {
+  url?: string;
+  target?: '_self' | '_blank' | '_parent' | '_top' | '_unfencedTop';
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):**  Maybe you could use the `type UrlTarget` you define in shared? Just so if we have a change there for some reason it will only happen in one place‏